config/sync/core.extension.yml

```diff
@@ -101,6 +102,7 @@ module:
   views: 10
   paragraphs: 11
   paragraphs_ee: 15
+  twigsuggest: 100
```

Comment:
Eh, I think this is a bit overkill.
I think it'd be better to just make a really simple `dpl_theme_suggestions` (or whatever) module, that adds the suggestions we need, that we can always expand on

Twigsuggest is only a "yellow" release candidate, and the code for it seems a bit.. complicated.

Reply:
:+1: I have replaced this with a hook implementation in the theme.